### DISCUSSION TITLE
Proposal to allow FTL until webrtc is implemented?

### DISCUSSION THREAD

**Discussion by TheTabKey** [obsproject/obs-studio#7092] [General]
I would like to propose to allow FTL until webrtc is implemented. Since flash is deprecated I do not like using rtmp since it is not viewable unless there is fragmentation (flv, etc). Webrtc/ftl just offer so much that I do not like with other protocols.

**Comment by dodgepong** [comment]
RTMP has nothing to do with Flash anymore in practice. We will continue to keep FTL deprecated due to the reasons outlined in this post: https://github.com/obsproject/obs-studio/discussions/4021
